single.php

```diff
@@ -21,6 +21,16 @@
 $post            = Timber::query_post( false, 'P4_Post' );
 $context['post'] = $post;
 
+// Strip Take Action Boxout block from the post content to add outside the normal block container.
+if ( strpos( $post->post_content, '[shortcake_take_action_boxout' ) ) {
```

Comment:
better check if strpos !== false, cause strpos could return 0 and this won't be evaluated

Reply:
Good point! Changed...